python/cucim/src/cucim/skimage/data/_binary_blobs.py

```diff
@@ -19,12 +25,10 @@ def binary_blobs(
     volume_fraction : float, default 0.5
         Fraction of image pixels covered by the blobs (where the output is 1).
         Should be in [0, 1].
-    seed : {None, int, `cupy.random.Generator`}, optional
-        If `seed` is None the `cupy.random.Generator` singleton is used.
-        If `seed` is an int, a new ``Generator`` instance is used,
-        seeded with `seed`.
-        If `seed` is already a ``Generator`` instance then that instance is
-        used.
+    rng : {`cupy.random.Generator`, int}, optional
+        Pseudo-random number generator.
+        By default, a PCG64 generator is used (see :func:`cupy.random.default_rng`).
+        If `rng` is an int, it is used to seed the generator.
```

Comment:
Do we still want to mention `seed` and just say it is deprecated (and to use `rng` instead)?

Note this comes up below as well. So any change here would be worth propagating

Reply:
Actually, the decorator also takes care of that by automatically appending Other Arguments with the deprecated parameter names...

it should look something like this
https://scikit-image.org/docs/stable/api/skimage.data.html#skimage.data.binary_blobs